erpnext/schools/doctype/student/student.json

```diff
@@ -165,7 +165,7 @@
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
-   "default": "Today", 
```

Comment:
Hi Neil,

It's a default value set to the date of creation of the record. However,
because the field is also set as readonly, there is no option to change it.
This effectively causes the enrollment date to always be the date on which
the record was created. My thought was that this was better represented as
a field that was free for entry but mandatory.

Regards,

Chude

On Aug 22, 2016 10:48, "Neil Trini Lasrado" <notifications@github.com>
wrote:

> In erpnext/schools/doctype/student/student.json
> <https://github.com/frappe/erpnext/pull/6128#discussion_r75648281>:
>
> > @@ -165,7 +165,7 @@
> >     "allow_on_submit": 0,
> >     "bold": 0,
> >     "collapsible": 0,
> > -   "default": "Today",
>
> This is just a default value. You can always change it. Its not Hard Coded
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frappe/erpnext/pull/6128/files/50994563b63078914a071b4c5feb659d545715e0#r75648281>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUJjzxZyyHo0b-zqCEqFy9RLlfI13Nyvks5qiXBMgaJpZM4JpM13>
> .
>